whipper/result/logger.py

```diff
@@ -65,7 +65,7 @@ def logRip(self, ripResult, epoch):
 
         # CD metadata
         lines.append("CD metadata:")
-        lines.append("  Album: %s - %s" % (ripResult.artist, ripResult.title))
+        lines.append("  Release: %s - %s" % (ripResult.artist, ripResult.title))
```

Comment:
Not sure about this, since this affects logger output and thus may be something we might want to not mess with, for any log parsers out there that can already read the logs?

Reply:
(Note that this line become more than 79 characters long, so the force-pushed commit changed this to be two lines, which is why this diff is now outdated, but the question still stands.)

Reply:
> Not sure about this, since this affects logger output and thus may be something we might want to not mess with, for any log parsers out there that can already read the logs?

It's all right: whipper still hasn't reached v1.0 ("first release for public use") 🤷

@itismadness ping.